Source/WebDriver/WebDriverMain.cpp

```diff
@@ -39,6 +39,6 @@ int main(int argc, char** argv)
     WebDriver::logChannels().initializeLogChannelsIfNecessary(WebDriver::logLevelString());
 #endif
 
-    WebDriver::WebDriverService service;
-    return service.run(argc, argv);
+    auto service = WebDriver::WebDriverService::create();
+    return service->run(argc, argv);
```

Comment:
Is this really needed?

Reply:
I added due to that guideline of not having public constructors, as WebDriverService is now RefCounted (more on that in the next comment).